backend/src/index.js

```diff
@@ -0,0 +1,32 @@
+require('dotenv').config();
+const express = require('express');
+const { Client } = require('pg');
+const app = express();
+
+const client = new Client({
+  connectionString: process.env.DATABASE_URL,
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+client.connect()
+  .then(() => console.log('Connected to the database'))
+  .catch(err => console.error('Connection error', err.stack));
+
+app.get('/items', async (req, res) => {
+  try {
+    const result = await client.query('SELECT * FROM items');
+    res.json(result.rows);
+  } catch (err) {
+    console.error(err);
+    res.status(500).send('Server error');
+  }
+});
```

Comment:
We may want to use pool instead of client, it would be basically the same but change the words to pool, add a maximum number, and a timeout. This enables us to have multiple concurrent requests (5) we could do more. 

```suggestion
const pool = new Pool({
  max: 5,
  idleTimeoutMillis: 50000,
  connectionString: process.env.DATABASE_URL,
  ssl: {
    rejectUnauthorized: false
  }
});

pool.connect()
  .then(() => console.log('Connected to the database'))
  .catch(err => console.error('Connection error', err.stack));

app.get('/items', async (req, res) => {
  try {
    const result = await pool.query('SELECT * FROM items');
    res.json(result.rows);
  } catch (err) {
    console.error(err);
    res.status(500).send('Server error');
  }
});
```

Reply:
good call out. That all looks good to me